src/imageupload.js

```diff
@@ -23,7 +24,7 @@ export default class ImageUpload extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ ImageUploadEngine ];
+		return [ ImageUploadEngine, ImageUploadProgress ];
```

Comment:
This way we disallow developers to have an upload button but different upload progress. We could move the button component to the separate plugin and make the main upload plugin only aggregates plugins, so do nothing but requires.